src/components/editor/Bindings/Backfill/index.tsx

```diff
@@ -0,0 +1,194 @@
+                setFormState({ status: FormStatus.UPDATING });
+
+                const singleBindingUpdate =
+                    bindingIndex > -1 && currentCollection;
+
+                const bindingMetadata: BindingMetadata[] = singleBindingUpdate
+                    ? [{ collection: currentCollection, bindingIndex }]
+                    : [];
+
+                updateBackfillCounter(
+                    draftSpec,
+                    increment,
+                    bindingMetadata
+                ).then(
+                    () => {
+                        const targetCollection = singleBindingUpdate
+                            ? currentCollection
+                            : undefined;
+
+                        setBackfilledCollections(increment, targetCollection);
+                        setFormState({ status: FormStatus.UPDATED });
+                    },
+                    (error) => {
+                        setFormState({
+                            status: FormStatus.FAILED,
+                            error: {
+                                title: 'workflows.collectionSelector.manualBackfill.error.title',
+                                error,
+                            },
+                        });
+                    }
+                );
+            }
+        },
+        [
+            bindingIndex,
+            currentCollection,
+            draftSpec,
+            evaluateServerDifferences,
+            setBackfilledCollections,
+            setFormState,
+            updateBackfillCounter,
+        ]
+    );
+
+    return (
+        <Box sx={{ mt: 3 }}>
+            <Stack direction="row" spacing={1} sx={{ mb: 2 }}>
+                <Stack spacing={1}>
+                    <Typography
+                        variant={
+                            bindingIndex === -1 ? 'formSectionHeader' : 'h6'
+                        }
+                        sx={{ mr: 0.5 }}
+                    >
+                        <FormattedMessage id="workflows.collectionSelector.manualBackfill.header" />
+                    </Typography>
+
+                    <Typography component="div">{description}</Typography>
+                </Stack>
+            </Stack>
+
+            <OutlinedToggleButton
+                value={value}
+                selected={selected}
+                disabled={
+                    formActive || !hasLength(collections) || allBindingsDisabled
+                }
+                onClick={(event, checked: string) => {
+                    event.preventDefault();
+                    event.stopPropagation();
+
+                    handleClick(checked === 'true' ? 'false' : 'true');
+                }}
+            >
+                <FormattedMessage id="workflows.collectionSelector.manualBackfill.cta.backfill" />
+
+                {selected ? (
+                    <Check style={{ marginLeft: 8, fontSize: 13 }} />
+                ) : null}
+            </OutlinedToggleButton>
```

Comment:
Since no content changes when this is enabled/disabled we should probably include _some_ aria/accessibility handling here. 

I think adding some aria content for the check to something like `enabled` would work.